nifi-nar-bundles/nifi-social-media-bundle/nifi-twitter-processors/pom.xml

```diff
@@ -35,6 +35,21 @@
             <artifactId>nifi-utils</artifactId>
             <version>1.17.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>com.twitter</groupId>
+            <artifactId>twitter-api-java-sdk</artifactId>
+            <version>1.1.2</version>
```

Comment:
This version can be incremented to `1.1.4`.

Reply:
Making the changes